pgmpy/estimators/BdeuScore.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+import numpy as np
+import pandas as pd
+from math import lgamma
+from pgmpy.estimators import StructureScore
+
+
+class BdeuScore(StructureScore):
```

Comment:
@chrisittner @ankurankan I feel it would be great if a reference section is also added to the class doc strings wherever it seems necessary.